packages/@expo/cli/src/lint/lintAsync.ts

```diff
@@ -0,0 +1,111 @@
+import JsonFile, { type JSONObject } from '@expo/json-file';
+import * as PackageManager from '@expo/package-manager';
+import spawnAsync from '@expo/spawn-async';
+import fs from 'fs/promises';
+import path from 'path';
+
+import { Log } from '../log';
+import { selectAsync } from '../utils/prompts';
+
+const WITH_PRETTIER = `module.exports = {
+  root: true,
+  extends: ["expo", "prettier"],
+  plugins: ["expo", "prettier"],
+  rules: {
+    "prettier/prettier": ["warn"],
+  },
+};
+
+`;
+
+const ESLINT_ONLY = `module.exports = {
+  root: true,
+  extends: ["expo"],
+};
+`;
+
+const setupLinting = async (projectRoot: string) => {
+  const result = await selectAsync(
+    'No eslint config found. Would you like to set up linting for this project?',
```

Comment:
```suggestion
    'No ESLint config found. Install and configure ESLint in this project?',
```

Reply:
i'd prefer for this to be a y/n prompt. then if they answer yes to installing eslint, we prompt to see if they want to install prettier too